### PR TITLE
Cleanup GSoC proposal

### PR DESCRIPTION
**Pull request by hegner**
Cleanup of last GSoC proposals. Using categories for filtering instead of posts

### CODE REVIEW

File: gsoc/cern.md [hunk 1]
```diff
@@ -8,5 +8,5 @@ layout: default
 ## Project Proposals
 
 {:.table .table-hover .table-striped}
-{% for post in site.categories.gsoc %}{% if post.organization == "CERN" %} | [ {{ post.title }} ]( {{ post.url }} ) | {% endif %} 
+{% for page in site.pages %}{% if page.categories contains 'gsocproposal' and page.organization == "CERN" %} | [ {{ page.title }} ]( {{ page.url }} ) | {% endif %}
```
Comment (jouvin) [inline gsoc/cern.md]: Why you don't do:
```
{% for page in site.gsoc %}{% if post.organization == "CERN" %} ...
```

File: gsoc/project_Geant4.md [hunk 2]
```diff
@@ -6,5 +6,5 @@ layout: plain
 ## Project Proposals
 
 {:.table .table-hover .table-striped}
-{% for post in site.categories.gsoc %}{% if post.project == "Geant4" %} | [ {{ post.title }} ]( {{ post.url }} ) | {% endif %}
```
Comment (jouvin) [inline gsoc/project_Geant4.md]: As above...

File: gsoc/project_ROOT.md [hunk 3]
```diff
@@ -6,5 +6,5 @@ layout: plain
 ## Project Proposals
 
 {:.table .table-hover .table-striped}
-{% for post in site.categories.gsoc %}{% if post.project == "ROOT" %} | [ {{ post.title }} ]( {{ post.url }} ) | {% endif %}
```
Comment (jouvin) [inline gsoc/project_ROOT.md]: Again...

File: gsoc/cern.md (hunk 1 above)
Comment (hegner) [inline gsoc/cern.md]: Because this kind of looping isn't supported for non-posts

Comment (jouvin) [inline gsoc/cern.md]: Yes it is. Look at https://github.com/HEP-SF/hep-sf.github.io/blob/master/_includes/navbar.ext#L20. This is one of the great features of collections. A great post IMO describing many of the advanced features of collections is http://ben.balter.com/2015/02/20/jekyll-collections/....

Comment (hegner) [inline gsoc/cern.md]: My hope was not relying on collections here. OK, will give it a try.

**Comment by hegner** [comment]
Using collections now

**Pull request merged**